apps/landingPage/src/components/Info.js

```diff
@@ -8,44 +8,101 @@ import styled from 'styled-components';
 import booking from '../images/bookings.png';
 import results from '../images/results.png';
 import search from '../images/search.png';
-import { documentation } from '../../../../linksConfig';
+import { documentation, github } from '../../../../linksConfig';
 import { BREAKPOINTS } from '../mediaQueriesConfig';
 
-export default function Info() {
-  return (
-    <Container id="features">
-      <Header>
-        <Heading>It's cross-platform</Heading>
-        <WrapperText>
-          <Text size="large" align="center">
-            PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered
-            yourTequila Tequila mixed with something else. Just grab
-            Margarita.You've had a look at Tequila, but you've always preffered
-            your Tequila mixed with something else.u've had a look at Tequ
-          </Text>
-        </WrapperText>
-        <Button type="secondary" width={125}>
-          <LinkNoStyle href={documentation}>Link to Docs</LinkNoStyle>
-        </Button>
-      </Header>
-      <ImagesAndGradientWrapper>
-        <BackgroundGradientAccent />
-        <PhoneImage src={search} alt="search" />
-        <PhoneImage src={results} alt="results" />
-        <PhoneImage src={booking} alt="booking" />
-      </ImagesAndGradientWrapper>
-    </Container>
-  );
+const text =
+  "PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered yourTequila Tequila mixed with something else. Just grabMargarita.You've had a look at Tequila, but you've always prefferedyour Tequila mixed with something else.u've had a look at Tequ";
+const phoneImageHeight = {
+  MOBILE: 500,
+  DESKTOP: 600,
+};
+const features = ['Search', 'Booking', 'MMB', 'Payments'];
+
+const renderCards = (features: Array<string>) => {
+  return features.map(feature => (
+    <CardWrapper key={feature}>
+      <Heading type="title2">{feature}</Heading>
+    </CardWrapper>
+  ));
+};
+
+type State = {|
+  +width: number,
+|};
+
+export default class Info extends React.Component<{||}, State> {
+  state = {
+    width: 0,
+  };
+
+  componentDidMount() {
+    this.handleWindowSizeChange();
+    window.addEventListener('resize', this.handleWindowSizeChange);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleWindowSizeChange);
+  }
+
+  handleWindowSizeChange = () => {
+    const width = typeof window !== 'undefined' ? window.innerWidth : 0;
+    this.setState({ width });
+  };
+
+  render() {
+    const alignment = this.state.width > BREAKPOINTS.TABLET ? 'left' : 'center';
```

Comment:
Also, if possible handle with css, if not, I see that this resize code is copy pasted. It could be extracted to a HOC.

Reply:
Should be better now, I've added `withSizeDetect` HOC